checkov/dockerfile/runner.py

```diff
@@ -9,12 +9,16 @@
 from checkov.dockerfile.registry import registry
 from checkov.runner_filter import RunnerFilter
 
-DOCKER_FILE_MASK = [DOCKERFILE_FILENAME]
+DOCKER_FILE_MASK = r"^(?:.+\.)?[Dd]ockerfile(?:\..+)?$"
 
 
 class Runner(BaseRunner):
     check_type = "dockerfile"
 
+    @staticmethod
+    def __is_docker_file(file):
```

Comment:
```suggestion
    def _is_docker_file(file):
```
one underscore is enough to symbolize this method is private, a double underscore does a bit more, which is called name mangling . This is only used to avoid naming conflicts with subclasses, which is not the case here.

Reply:
Learned something new today. Will definitely change this!